src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareString = (a, b) => {
+    return +(String(a) > String(b));
+  };
```

Comment:
Your `compareString` function now works unproperly.  It should also return `-1` in case when `a <  b`.
Please rewrite it using `if else`.
First screen - how it should work, second - how it work now.
![Screenshot 2022-08-17 at 10 58 48](https://user-images.githubusercontent.com/17098736/185068894-168a8f2d-9427-4973-b438-9b2f2db7df5d.png)
![Screenshot 2022-08-17 at 11 00 15](https://user-images.githubusercontent.com/17098736/185068899-74b4de8d-4ea0-4bb7-8f10-1130f9d0733a.png)

Reply:
fixed it